test/hotspot/jtreg/vmTestbase/gc/gctests/PhantomReference/phantom001/phantom001.java

```diff
@@ -54,22 +53,23 @@
  *
```

Comment:
Pre-existing: There doesn't seem to be any code for check (6) - reference.clear() doesn't throw.  You fixed this in some of the tests, but not this one.

Reply:
I removed this comment. We don't test clear().